app/controllers/clearance/sessions_controller.rb

```diff
@@ -33,4 +40,8 @@ def url_after_create
   def url_after_destroy
     sign_in_url
   end
+
+  def url_for_signed_in_users
+    url_after_create
```

Comment:
I think it's possible we may want to clarify `Clearance.configuration.redirect_url` in the future, but I don't think it's something we have to do right now.

>  I think the reason for this PR is to be able to have two different redirect URLs depending on if you are a newly created user, or a user who has already logged in. Oh wait...maybe I'm reading this wrong. My thought was that I want to be able to specify a different redirect URL for a newly created user (taking them to a "fill out additional profile information" page). But maybe create in this context means the creation of a new session (i.e. after sign in v.s. visiting sign in page then automatically redirecting if you're signed in).

This is the sessions controller, not the users controller. The intent of this PR is to prevent already signed in users from reaching the log in page, which might be confusing. Though that raises an interesting question -- do we have this problem on the new user page as well?